systemtest/src/main/java/io/strimzi/systemtest/HttpBridgeBaseST.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.fabric8.kubernetes.api.model.Service;
+import io.strimzi.systemtest.utils.StUtils;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.ext.web.codec.BodyCodec;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static io.strimzi.systemtest.Resources.getSystemtestsServiceResource;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Base for test classes where HTTP Bridge is used.
+ */
+public class HttpBridgeBaseST extends MessagingBaseST {
+    private static final Logger LOGGER = LogManager.getLogger(HttpBridgeBaseST.class);
+    private WebClient client;
+
+    protected String brdigeExternalService = CLUSTER_NAME + "-bridge-external-service";
+
+    @BeforeAll
+    void prepareEnv(Vertx vertx) {
+        // Create http client
+        client = WebClient.create(vertx, new WebClientOptions()
+                .setSsl(false)
+                .setTrustAll(true)
+                .setVerifyHost(false));
```

Comment:
why setting these properties related to SSL if you are using SSL=false so verify host and trust all aren't needed?

Reply:
TBH I am not sure where I get it, I will remove it and push fixes for other typos